src/v1/Models/Item.php

```diff
@@ -62,6 +64,31 @@ public static function boot()
     });
   }
 
+  public static function booted()
+  {
+    parent::booted();
+    static::created(function ($model)
+    {
+      // check if type is a tree
+      $type = \App\v1\Models\Config\Type::find($model->type_id);
+      if ($type->tree)
+      {
+        // It's a tree, manage this
+        // get the item created
+        // get the parent item
+        // update treepath in this item
+        $currItem = (new self())->find($model->id);
```

Comment:
Can you add a comment to explain that we need to reload the model from the database to get the value of `id_bytype`? It took me some time to understand why you do this.

Reply:
I added more explainations ;)